Makefile

```diff
@@ -6,10 +6,10 @@ no-debug ?= ## No symbolic debug info
 verbose ?= ## Run specs in verbose mode
 link-flags ?= ## Additional flags to pass to the linker
 
-OS := $(shell uname -s | tr '[:upper:@]' '[:lower:]')
+OS := $(LC_CTYPE=C $(shell uname -s | tr '[:upper:]' '[:lower:]'))
```

Comment:
I had to add an environment variable to get it to work correctly in my environment.